frontEnd/showError.ts

```diff
@@ -0,0 +1,48 @@
+export let error = document.createElement("td");
+error.setAttribute("class", "invalid");
+class ShowError{
+    passPhone(phone:HTMLInputElement){
+        let phoneField = phone.parentNode?.parentNode as HTMLTableRowElement;
```

Comment:
redundant code. all functions are doing same work. only fields and error text is diff. make a common function.